src/main/java/racingcar/refactoring/domain/GameResult.java

```diff
@@ -0,0 +1,32 @@
+package racingcar.refactoring.domain;
+
+import java.util.List;
+
+public class GameResult {
+
+    private final List<RoundResult> roundResults;
+
+    public GameResult(List<RoundResult> roundResults) {
+        this.roundResults = roundResults;
+    }
+
+    public List<RoundResult> getRoundResults() {
+        return roundResults;
+    }
+
+    public List<String> findWinners() {
+        return createWinnersName();
+    }
+
+    private List<String> createWinnersName() {
+        return findFinalRoundResult().findNamesByPosition(getMaxPosition());
+    }
+
+    private int getMaxPosition() {
+        return findFinalRoundResult().findMaxPosition();
+    }
+
+    private RoundResult findFinalRoundResult() {
+        return roundResults.get(roundResults.size() - 1);
+    }
```

Comment:
```suggestion
    private List<String> createWinnersName() {
        RoundResult finalResult = roundResults.get(roundResults.size() - 1);
        return finalResult.findMaxPositionNames();
    }
```
`findFinalRoundResult`를 변수로 뽑아내면 될 것 같은데 불필요한 메소드 호출이 두번 발생하고 있는 것 같아요. 변수를 사용하는 것은 가독성을 떨어뜨리는 것은 아니며 오히려 메소드 체이닝만을 활용하면 가독성이 더 떨어질 수 있습니다.

그리고 `findMaxPosition`은 어짜피 `RoundResult`내부에 있으니 외부에서 파편화되기보다는 maxPosition은 `findMaxPositionNames` 내부에서 계산하여 사용하여도 될 것 같네요 :)